admin/src/main/resources/application.yml

```diff
@@ -14,6 +14,11 @@ app:
           client-secret: password
           refresh-token-validity: ${COMMON_REFRESH_TOKEN_VALIDITY:1209600}
           resource-ids: authorization
+  feign:
+    appery-io:
+      url: https://api.appery.io/rest/1/db/collections
+      database:
+        id: ${APPERY_API_KEY:example}
```

Comment:
if the APPERY_API_KEY is not set - what will be? 
I assume that in this case we need to disable statistics. But if you set it to "example" - probably it will try to send request with value "example"

Reply:
![image](https://user-images.githubusercontent.com/64147071/102239355-d40ee900-3f07-11eb-8542-f3168adbb729.png)
![image](https://user-images.githubusercontent.com/64147071/102239403-e12bd800-3f07-11eb-8b60-a7a0d8d638f6.png)
if APPERY_API_KEY  is not set, than value is "" in spite of -  :examle